notebooks/Using_Cache.ipynb

```diff
@@ -432,7 +432,7 @@
     "\n",
     "#### Cache Statistics\n",
     "\n",
-    "If used in the multi-processing environment (e.g, using `concurrent.futures.ProcessPoolExecutor()`), cache hit count (`hit_count`) and miss count (`miss_count`) wouldn't be recorded in the main process's cache object.\n",
+    "If used in the multi-processing environment (e.g, using `concurrent.futures.ProcessPoolExecutor()`), cache hit count (`hit_count`) and miss count (`miss_count`) wouldn't be recorded in the release/25.12 process's cache object.\n",
```

Comment:
We recently noticed that this text was changed as part of branching. However it shouldn't have been changed. Is there something we can do in the branching logic to avoid this in the future?